htdocs/editor_tab.js

```diff
@@ -1402,7 +1445,7 @@ var editor = function () {
                 }
                 if(opts.toggle) whither = 'hide';
             }
-            update_history_nodes(node, whither, null)
+            return update_history_nodes(node, whither, null)
```

Comment:
This turned out not to be necessary with the refactor, since now we don't have to show the menu in order to act on it. It's still a good idea to return promises wherever they are generated, even if they're not currently used.